web/src/ui/roomlist/RoomList.css

```
@@ -106,22 +106,28 @@ div.room-entry {
 		width: 3rem;
 
 		> div.unread-count {
+			--unread-count-size: 1rem;
+			--unread-count-padding-inline: calc(1rem/4);
+			--unread-count-padding-block: calc(1rem/8);
```

using `calc` with a constant value is a bit silly

```suggestion
			--unread-count-padding-inline: .25rem;
			--unread-count-padding-block: .125rem;
```

---

unless it was meant to be based on unread-count-size, in which case

```suggestion
			--unread-count-padding-inline: calc(var(--unread-count-size)/4);
			--unread-count-padding-block: calc(var(--unread-count-size)/8);
```